### ISSUE TITLE
Android phone and Fire Tablet do not work with Pi-Hole

### ISSUE THREAD

**Issue by BWBama85** [pi-hole/pi-hole#2273]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*.
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the the source code relevant to this issue?:**

1

---
**Expected behaviour:**

Pi-hole should work on Android devices connected via Wifi

**Actual behaviour:**

Pi-hole does not block any requests on Android

**Steps to reproduce:**

I have the following network setup:

Cable Modem plugged into Edgerouter X running latest firmware (192.168.1.1)
Two Pi-Holes (192.168.1.42, 192.168.1.190) plugged into Edgerouter X using Google Upstream DNS for IPV4 and IPV6. 
Linksys WRT1900ACS plugged into Edgerouter X in bridge mode

Edgerouter X config:

https://imgur.com/a/AfOf8H1

After changing Nameservers and DHCP DNS I ran the following command via cli in Edgerouter X:

```
configure
set service dns forwarding system
commit
save
```

I confirm that my phone is pointing to 192.168.1.1 via Network Info II:

https://imgur.com/a/Fnt1uGt

**Debug token provided by [uploading `pihole -d` log](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug):**

Pi-hole 192.168.1.42:  kzpsy1ydfi
Pi-hole 192.168.1.190: 0nfm4gwy7g

**Troubleshooting undertaken, and/or other relevant information:**

I've reset the router and pi-holes multiple times and configured them exactly as described above. Both my Fire Tablet and Android phones do not work with Pi-hole and ads are shown.

**Comment by technicalpyro** [comment]
where is DHCP being done on the edgerouter or the linksys in both places you need to make sure that the piholes are assigned to be the only DNS there is a chance if you have only assigned it in the Edgerouter that it is not being properly passed to the linksys and on through to your wireless devices

**Comment by BWBama85** [comment]
The Linksys router does not handle DHCP. It is in Bridge Mode. DHCP is disabled in both Pi-holes.

![linksys-bridge-mode](https://user-images.githubusercontent.com/41128321/42601322-e1200c76-852a-11e8-85f5-a6520bb80623.PNG)

**Comment by BWBama85** [comment]
I want to point out, that other WiFi devices connect to Pi-hole correctly. For example, my Fire TV stick and my MacBook Pro.

**Comment by technicalpyro** [comment]
this [thread ](https://community.linksys.com/t5/Wireless-Routers/How-to-configure-1900AC-as-an-Access-Point-NOT-Bridge/td-p/871359)may have some insight for you

**Comment by BWBama85** [comment]
Thanks. I like it as a bridge because I also use it as a dumb switch for a Windows 10 server. I do not think it is the router as my Laptop and other WiFi devices are working properly with Pi-Hole. It seems specific to Android devices only.

**Comment by BWBama85** [comment]
Ok, I have found the issue on my Pixel 2 XL. I am running Android P beta and switched the Private DNS from Automatic to Off and restarted. Now my Pixel 2 XL is working correctly with Pi. I will open a new ticket for that. Will leave this one open while I continue to figure out what is wrong with my Fire Tablet. It may be just that it is ancient...

**Comment by technicalpyro** [comment]
yeah android P would cause that i have the 2XL myself and couldnt recreate so went to google as for yuour firestick are you able to set the DNS manually? if so set it to you piholes

**Comment by BWBama85** [comment]
I'll just manually set the DNS for my tablet. Not that big of a deal. It never leaves home. I opened this ticket: https://github.com/pi-hole/pi-hole/issues/2274 for the Android P issue. As of now, Automatic is the default setting so when it is fully released could cause a lot of issues to be opened.
